core/src/Streamly/Unicode/Stream.hs

```diff
@@ -75,18 +75,24 @@
 --
 module Streamly.Unicode.Stream
     (
-    -- * Construction (Decoding)
-      decodeLatin1
+      DecodeState
+    , CodePoint
+    , DecodeError(..)
+
+    -- * Resumable UTF-8 decoding
```

Comment:
Only `resumeDecodeUtf8Either`. None of the others are resumable. We should remove the rest of the API from this heading. Only `resumeDecodeUtf8Either` should be under the resumable heading.